templates/engage/casestudy-sensape.html

```diff
@@ -0,0 +1,30 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Sensape uses Landscape to remotely support revolutionary interactive retail displays{% endblock %}
+
+{% block meta_description %}Sensape is teaching computers to see, understand and interact with people at trade events and in retail locations. Instead of using unchanging, predefined content, the Sensape systems senses its environment and reacts accordingly.{% endblock %}
+
+{% block meta_copydoc %}https://app-sjg.marketo.com/#LP5880A1LA1{% endblock meta_copydoc %}
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="Sensape uses Landscape to remotely support revolutionary interactive retail displays" image="811c3501-Sensape_Logo_rgb_Animated.gif" url="#the-form" cta="Download the case study" %}
+
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="col-7">
+        <p>Sensape is teaching computers to see, understand and interact with people at trade events and in retail locations. Instead of using unchanging, predefined content, the Sensape systems senses its environment and reacts accordingly.</p> 
```

Comment:
please use 2 spaces indentation instead of 4 as this is a Canonical best practice https://canonical-web-and-design.github.io/practices/coding/html.html